src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -1430,30 +1429,32 @@ uct_rc_mlx5_iface_common_copy_to_dm(uct_rc_mlx5_dm_copy_data_t *cache, size_t hd
                                     const void *payload, size_t length, void *dm,
                                     uct_ib_log_sge_t *log_sge)
 {
-    size_t head      = (cache && hdr_len) ? ucs_min(length, sizeof(*cache) - hdr_len) : 0;
-    size_t body      = ucs_align_down(length - head, sizeof(uint64_t));
-    size_t tail      = length - (head + body);
-    uint64_t *dst    = dm;
-    uint64_t padding = 0; /* init by 0 to suppress valgrind error */
-    int i            = 0;
-    typedef uint64_t misaligned_uint64_t UCS_V_ALIGNED(1);
+    typedef uint64_t aligned_t;
+    typedef aligned_t unaligned_t UCS_V_ALIGNED(1);
+
+    aligned_t padding = 0; /* init by 0 to suppress valgrind error */
```

Comment:
could be just uint64_t, we don't access padding by pointer anyway

Reply:
I just wanted to set all variables consistent type: padding type should be same with aligned datatype.
but ok, fixed